src/app/GitCommands/Git/GitModule.cs

```diff
@@ -97,7 +97,7 @@ public GitModule(string? workingDir)
             _wslDistro = AppSettings.WslGitEnabled ? PathUtil.GetWslDistro(WorkingDir) : "";
             if (!string.IsNullOrEmpty(_wslDistro))
             {
-                _gitExecutable = new Executable(() => AppSettings.WslGitCommand, WorkingDir, $"-d {_wslDistro} {AppSettings.WslGitPath} ");
+                _gitExecutable = new Executable(() => AppSettings.WslGitCommand, WorkingDir, $"-d {_wslDistro} --cd {WorkingDir.Quote()} {AppSettings.WslGitPath} ");
```

Comment:
A comment that some WSL need the "--cd" will be useful for future readers here and in FormProcess.